wikidict/lang/defaults.py

```diff
@@ -78,10 +78,6 @@ def last_template_handler(
         text = parts[1] if len(parts) == 2 else word
         return transliterate(lang, text)
 
-    # {{tpl|item}} -> <i>(Templatet gf)</i>
-    if len(template) == 2:
-        return term(capitalize(lookup_italic(tpl, locale)))
```

Comment:
I believe the best way would be to call lookup_italic(tpl, locale, True): here, just like above.
Then the template will be reported by the MISSING_TPL_SEEN code.